components/layout/fragment.rs

```diff
@@ -2602,11 +2602,6 @@ impl Overflow {
         self.paint = self.paint.union(&other.paint);
     }
 
-    pub fn union_rect(&mut self, rect: &Rect<Au>) {
```

Comment:
I'm not sure whether we want to remove it entirely, or add a `#[allow(dead_code)]` above it instead.